### PR TITLE
[TFLITE][FRONTEND]Reduce_any op parsing support

### PR DESCRIPTION
**Pull request by siju-samuel**
@FrozenGene @u99127 @kevinthesun Please help to review. Thanks.

### CODE REVIEW

File: tests/python/frontend/tflite/test_forward.py [hunk 1]
```diff
@@ -1099,11 +1099,19 @@ def _test_reduce_sum(data, keep_dims=None):
...
+    return _test_reduce(math_ops.reduce_any, data, keep_dims)
 
-def _test_forward_reduce(testop):
+
+def _test_forward_reduce(testop, dtype="float32"):
     """ Reduce """
-    data0 = [np.random.rand(16, 16, 16, 16).astype("float32"), None]
-    data1 = [np.random.rand(16, 16, 16, 16).astype("float32"), np.array([1, 2], dtype=np.int32)]
+    data0 = [np.random.rand(16, 16, 16, 16).astype(dtype), None]
+    data1 = [np.random.rand(16, 16, 16, 16).astype(dtype), np.array([1, 2], dtype=np.int32)]
```
Comment (wyc-ruiker) [inline tests/python/frontend/tflite/test_forward.py]: I'm a little confused. I think the elements of `np.random.rand().astype("bool")`  will be 1 in most cases.

File: tests/python/frontend/tflite/test_forward.py [hunk 2]
```diff
@@ -1099,11 +1099,22 @@ def _test_reduce_sum(data, keep_dims=None):
...
+#######################################################################
+# Reduce_any
+# -----------
+
+def _test_reduce_any(data, keep_dims=None):
+    """ One iteration of reduce_any """
+    return _test_reduce(math_ops.reduce_any, data, keep_dims)
 
-def _test_forward_reduce(testop):
+
```
Comment (wyc-ruiker) [inline tests/python/frontend/tflite/test_forward.py]: unnecessary line feed.

File: tests/python/frontend/tflite/test_forward.py [hunk 3]
```diff
@@ -1099,11 +1099,22 @@ def _test_reduce_sum(data, keep_dims=None):
...
+def _test_forward_reduce(testop, dtype=None):
     """ Reduce """
     data0 = [np.random.rand(16, 16, 16, 16).astype("float32"), None]
     data1 = [np.random.rand(16, 16, 16, 16).astype("float32"), np.array([1, 2], dtype=np.int32)]
+    if dtype == 'bool':
+        data0[0] = (data0[0] < 0.5).astype(dtype)
+        data1[0] = (data1[0] < 0.5).astype(dtype)
```
Comment (wyc-ruiker) [inline tests/python/frontend/tflite/test_forward.py]: It seems a little strange. Wait for other review's opinions.

Comment (inadob) [reply tests/python/frontend/tflite/test_forward.py]: To generate boolean numpy boolean data you can use 
`data = np.random.choice(a=[False, True], size=(1, 2, 3)).astype('bool')`

**Comment by inadob** [comment]
LGTM

**Comment by FrozenGene** [comment]
Thanks @siju-samuel @inadob @wyc-ruiker merged

**Pull request merged**